src/cloud-sql-instance.ts

```diff
@@ -152,6 +171,14 @@ export class CloudSQLInstance {
       this.next = undefined;
       return Promise.reject('closed');
     }
+    if (this?.instanceInfo?.domainName && !this.checkDomainID) {
```

Comment:
let's add a comment about why we are lazy instantiating ticker.

Reply:
Done.